### PR TITLE
Add code examples of close event

### PR DESCRIPTION
**Pull request by nononokam**
I would like to add code examples of close event to explainer.
This is the [context](https://groups.google.com/a/chromium.org/g/blink-dev/c/enQs5VEdOmU/m/gFFd75aXBgAJ).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,12 @@ when an entangled `MessagePort` is closed.
 Given a pair of entangled ports, `portA` and `portB`,
 if `portA` is closed,
 the event is fired on `portB`.
+this `close` event is used as follows:
```
Comment (fergald) [inline README.md]: Capital T for "this".

File: README.md [hunk 2]
```diff
@@ -155,6 +155,12 @@ when an entangled `MessagePort` is closed.
...
+```
+portB.onclose = () => {
+    console.log("portA has been closed.");
+}
```
Comment (fergald) [inline README.md:163]: Maybe add something like 

```
// Free up resources associated with the port.
...
```

File: README.md [hunk 3]
```diff
@@ -155,6 +155,12 @@ when an entangled `MessagePort` is closed.
 Given a pair of entangled ports, `portA` and `portB`,
 if `portA` is closed,
 the event is fired on `portB`.
+this `close` event is used as follows:
+```
+portB.onclose = () => {
+    console.log("portA has been closed.");
```
Comment (fergald) [inline README.md]: indent 2 spaces.

**Review by fergald — CHANGES_REQUESTED** [review]

**Comment by nononokam** [comment]
Thank you for review! I worked on your comments.

**Review by fergald — APPROVED** [review]
Thanks!

**Pull request merged**